### ISSUE TITLE
Submitting instructions improvement

### ISSUE THREAD

**Issue by schoeke** [rdpeng/RepData_PeerAssessment1#107]
When the task is submitted, it ceased to be a template. It would be cleaner to rename the file to PA1. The file name should convey the purpose.

**Comment by ghost** [comment]
Agreed. Specifying some unique identify information in the markdown before submission would also be helpful.
